### ISSUE TITLE
.riload() with watch mode enabled doesn't fire

### ISSUE THREAD

**Issue by acarabott** [tubalmartin/riloadr#16]
Calling `.riload()` on a group after AJAX loading of new elements doesn't fire with `watchViewportWidth: true` and `breakpoint` and `prevBreakpoint` are equal.

This seems to be be remedied with the following, but I'm not 100% confident about the robustness here. 

```
if (! watchViewportEnabled ||
    watchViewportEnabled && (
        ! prevBreakpoint || (
        watchViewportUp && isBreakpointWider(breakpoint, prevBreakpoint) ||
        watchViewportBoth && ! areBreakpointsEqual(breakpoint, prevBreakpoint) ||
        update
    ))) {
```

**Comment by mnakalay** [comment]
Hello,
I don't think that option is supposed to get a value of TRUE.
It's either "wider" or "*"
The first one monitors only widening of the viewport and the second checks both ways.

**Comment by acarabott** [comment]
Apologies I meant when watchViewportEnabled is `true`, which happens when `watchViewportWidth: '*'`

**Comment by tubalmartin** [comment]
Fixed in 1.5.1! Thank you very much for reporting! I missed that, good catch!!
